app/src/main/java/com/pps/globant/fittracker/mvp/presenter/LoginLocallyPresenter.java

```diff
@@ -0,0 +1,86 @@
+package com.pps.globant.fittracker.mvp.presenter;
+
+import android.app.Activity;
+import android.content.Intent;
+
+import com.pps.globant.fittracker.FirstAppScreenActivity;
+import com.pps.globant.fittracker.R;
+import com.pps.globant.fittracker.SignInFormActivity;
+import com.pps.globant.fittracker.mvp.model.DataBase.UsersRepository;
+import com.pps.globant.fittracker.mvp.model.LoginLocallyModel;
+import com.pps.globant.fittracker.mvp.view.LoginLocallyView;
+import com.pps.globant.fittracker.utils.BusProvider;
+import com.squareup.otto.Subscribe;
+
+public class LoginLocallyPresenter {
+    private LoginLocallyView view;
+    private LoginLocallyModel model;
+    private Activity activity;
```

Comment:
remove this, you need use view.getActivity()

Reply:
done